pushkin/request/event_handlers.py

```diff
@@ -87,16 +88,30 @@ def __init__(self, event_id, message_ids):
         self.message_ids = message_ids
 
     def handle_event(self, event, event_params):
+
+        # Allowed characters for parameter name are [a-zA-Z0-9_]
+        def get_parameter(param_name):
+            parameter = event_params.get(param_name)
+            if type(parameter) == unicode:
+                parameter = parameter.encode('utf-8')
+            return parameter
+
         raw_messages = []
         if self.event_id == event.event_id:
             for message_id in self.message_ids:
                 try:
                     localized_message = database.get_localized_message(event.user_id, message_id)
+                    parameter_names = [key.group(0).strip("{}") for key in
+                                         re.finditer("\{[a-zA-Z0-9_]+\}", localized_message.message_text)]
```

Comment:
Think about using re.compile outside the function, that way we separate regex definition from it's usage.